packages/ui-extensions/src/surfaces/point-of-sale/render/components/Stack/Stack.ts

```diff
@@ -1,61 +1,103 @@
 import {createRemoteComponent} from '@remote-ui/core';
-import type {HorizontalSpacing, VerticalSpacing} from '../Spacing/Spacing';
-
-export type Spacing =
-  | 0.5
-  | 1
-  | 2
-  | 3
-  | 4
-  | 5
-  | 6
-  | 7
-  | 8
-  | 9
-  | 10
-  | 11
-  | 13
-  | 16;
-
-export interface StackProps {
-  /**
-   * The direction of the stack.
-   */
-  direction: 'vertical' | 'horizontal';
-  /**
-   * The alignment of the children along the cross axis.
-   * @defaultValue 'flex-end'
-   */
-  alignment?:
-    | 'flex-end'
-    | 'center'
-    | 'flex-start'
-    | 'space-around'
-    | 'space-between'
-    | 'space-evenly';
+import type {PaddingProps, SizingProps} from '../shared/box';
+import {
+  ContentDistribution,
+  ContentPosition,
+  SpacingKeyword,
+  GapProps,
+} from '../shared/inner-layouts';
+import {
+  HorizontalSpacing,
+  Spacing,
+  VerticalSpacing,
+} from '../shared/deprecated-types';
+
+/**
+ *
+ * @deprecated Use the `block` or `inline` as a value instead.
+ */
+type DeprecatedStackDirection = 'vertical' | 'horizontal';
```

Comment:
In the end I included these in direction, but there's no way to separate mark these as deprecated that would cross out vertical or horizontal in the idea which is really lame.